polaris-core/src/main/java/org/apache/polaris/core/auth/PolarisAuthorizerImpl.java

```diff
@@ -177,7 +177,6 @@ public class PolarisAuthorizerImpl implements PolarisAuthorizer {
     SUPER_PRIVILEGES.putAll(
         NAMESPACE_LIST,
         List.of(
-            CATALOG_MANAGE_CONTENT,
```

Comment:
I think this is effectively achieving the opposite: you are _denying_ `NAMESPACE_LIST` to principals having `CATALOG_MANAGE_CONTENT`.